amundsen_application/static/js/components/common/PopularTables/tests/index.spec.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+
+import {
+  POPULAR_TABLES_INFO_TEXT,
+  POPULAR_TABLES_LABEL,
+  POPULAR_TABLES_SOURCE_NAME,
+} from '../constants';
+import InfoButton from 'components/common/InfoButton';
+import SearchList from 'components/SearchPage/SearchList';
+import globalState from 'fixtures/globalState';
+import { PopularTables, PopularTablesProps, mapStateToProps, mapDispatchToProps } from '..';
+
+describe('PopularTables', () => {
+  const setup = (propOverrides?: Partial<PopularTablesProps>) => {
+    const props: PopularTablesProps = {
+      popularTables: jest.fn() as any,
+      getPopularTables: jest.fn(),
+      ...propOverrides
+    };
+    const wrapper = shallow<PopularTables>(<PopularTables {...props} />)
+
+    return { props, wrapper };
+  };
+  let wrapper;
+  let props;
+  
+  describe('componentDidMount', () => {
+    let getPopularTablesSpy;
+    beforeAll(() => {
+      const setupResult = setup();
+      wrapper = setupResult.wrapper;
+      props = setupResult.props;
+      getPopularTablesSpy = jest.spyOn(props, 'getPopularTables');
```

Comment:
😮 Today I learned testing this method with a default case will still work without having to call `wrapper.instance().componentDidMount();` before the test.